pkg/client/client.go

```diff
@@ -1015,6 +1054,51 @@ func monitorKeepTerminatedEvent(
 					networkProvider,
 					hostChain.Signing().PublicKeyToAddress,
 				)
+				if err != nil {
+					logger.Errorf(
+						"failed to retrieve btc recovery addresses on keep [%s]: [%v]",
```

Comment:
Additionally to the address, we want to determine `maxFeePerVByte`. Maybe we could rephrase this error message to something like "failed to establish recovery details for keep ...." (please feel free to find a better word instead of `establish`).